src/teaching-element/Assessment/SingleChoice.vue

```diff
@@ -36,20 +38,30 @@ export default {
     return { userAnswer: this.submission };
   },
   computed: {
+    mappedAnswers() {
```

Comment:
This is not used anywhere. I assume it should be used in the `v-for` directive instead of the `answers` prop.

Reply:
Good catch 👍